healvis/data/configs/obsparam_test.yaml

```diff
@@ -13,9 +13,8 @@ telescope:
   telescope_location: (-30.72152777777791, 21.428305555555557, 1073.0000000093132)  # LatLonAlt location
   telescope_name: HERA
 freq:
-  Nfreqs: 3
-  start_freq: 100000000.0  # Hz
-  bandwidth:  100000000.0  # Hz
+  freq_array: [1.00e+08, 1.05e+08, 1.10e+08, 1.15e+08, 1.20e+08,
+                1.25e+08, 1.30e+08, 1.35e+08, 1.40e+08, 1.45e+08]
 time:
   Ntimes: 5
   time_cadence: 100.0  # seconds
```

Comment:
also idk if you noticed this, but I switched our vocabular from `integration_time` to `time_cadence`. This is perhaps what you were referring to in "what does a bandwidth [or integration similarly] mean for a simulator?" in that we aren't _integrating_ data as the correlator is doing, we are _sampling_. For that reason I didn't like the usage of `integration_time` to set the time_array spacing, so I opted to use `time_cadence` instead, and now the `integration_time` array in the output `UVData` is just set to one by default.

Reply:
I agree, it's a much better term for what we're doing. The change in terminology should be finished throughout.